obj.js

```diff
@@ -224,7 +224,7 @@ async function funcListener(payload) {
   // may throw
   let result = obj[name](...args);
 
-  return await mapOutgoingObjects(result);
+  return await mapOutgoingObjects(await result);
```

Comment:
Before awaiting, check whether it's a `Promise`. `await` on a value is *not* a no-op.

Reply:
This already landed like this on `master`, because it was part of PR#1.